### ISSUE TITLE
Ensure km and kg are read out as kilometers and kilograms

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#95]
The abbreviations km and kg are read out correctly in the the following browser/AT combos:
- [ ] Windows 10 + Latest Chrome + Latest JAWS
- [ ] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver

**Comment by zepumph** [comment]
We should determine not only if the abbreviations work in GFL(B), we should determine if they work in aria valuetext, other basic html (like `<p>`), and alerts, or just in aria-valuetext attributes.

The goal is to be consistent across this sim, and future ones.

**Comment by jessegreenberg** [comment]
Here is a JSFiddle to test with a number of contexts: https://jsfiddle.net/068hje1w/3/

**Comment by jessegreenberg** [comment]
For all of the tests in that JSFiddle, JAWS says "kg" and never kilograms. I tested in both Chrome and FF. I also tested the sim and JAWS says 'kg' there too like "4 billion kg".

**Comment by jessegreenberg** [comment]
Here is a test with everything swapped to km, and JAWS also read 'km' eveywehre instead of 'kilometers' https://jsfiddle.net/krnofdj2

**Comment by jessegreenberg** [comment]
But interesting notes:

JAWS has "Expand Abbreviations" and "Expand Acronyms" settings. I enabled both but it still just read "km" and "kg".

JAWS has a dictionary where you can define words and how they are read. I added `km=kilometers`, and it read all usages of "km" in the above JSFiddle as "kilometers". I don't think that asking our users to modify the JAWS dictionary is acceptable, but it could be useful.

Hope this helps.

**Comment by terracoda** [comment]
@jessegreenberg  for that "Expand Abbreviations" setting to work, JAWS might need the abbreviation coded as an abbreviation like so: `<abbr>km</abbr>` 

I think the other tag,  <acronym>,  was deprecated.

If JAWS can't read "km" as "kilometers" without having a user updates its dictionary, I think we should stick with words in the sim's content.

**Comment by jessegreenberg** [comment]
Oh thanks @terracoda , Ill try that in a test.

**Comment by jessegreenberg** [comment]
I tried with this markup and still only received "km" from JAWS in all contexts https://jsfiddle.net/71zqfgmp

**Comment by terracoda** [comment]
@jessegreenberg, I think that JAWS might be notoriously bad at reading abbreviations. 
I seem to remember examples from other folks, a periodic table comes to mind.

For this GFL Regular and Basics, I think we should stick with the spelled out words for now.

**Comment by terracoda** [comment]
@zepumph, can you do a search and replace for all the abbreviations?

For REGULAR:
- Any instance of `m` should be `meters`
- Any instances `kg` should be `kilograms`

For BASICS:
- Any instance of `km` should be `kilometers`
- Any instances `kg` should be `kilograms`
- Any instances of `billions of kg` should be `billions of kilograms`

**Comment by terracoda** [comment]
@zepumph, I think this one is easy, so I am marking it as top priority just to get it out of the way ;-)

**Comment by terracoda** [comment]
@zepumph, apologies for the multiple re-arrangement of priorities. 
Do the Distance issues first, then this one.

**Comment by zepumph** [comment]
I updated all the usages in a11y strings. I wanted to double check with you about the scope you were recommending. Currently there is displayed text with the abbreviations, and to this point a11y may have used these strings. 

We could either update the visual text to use the full word, or we could go through and make sure that a11y related units are totally separate from the visual ones. Honestly I'm not really sure how much refactoring this would take. I poked around for a bit and didn't see a ton of overlap, but I would want to be more thorough. 

What do you think?

**Comment by terracoda** [comment]
@zepumph looks good closing.
